test/e2e/plain_provisioner_test.go

```diff
@@ -297,7 +292,56 @@ var _ = Describe("plain provisioner bundle", func() {
 				WithTransform(func(c *metav1.Condition) string { return c.Type }, Equal(rukpakv1alpha1.TypeUnpacked)),
 				WithTransform(func(c *metav1.Condition) metav1.ConditionStatus { return c.Status }, Equal(metav1.ConditionFalse)),
 				WithTransform(func(c *metav1.Condition) string { return c.Reason }, Equal(rukpakv1alpha1.ReasonUnpackFailed)),
-				WithTransform(func(c *metav1.Condition) string { return c.Message }, ContainSubstring(`open manifests: file does not exist`)),
+				WithTransform(func(c *metav1.Condition) string { return c.Message }, Equal(`get objects from bundle manifests: readdir manifests: file does not exist`)),
```

Comment:
nit: this check of check still feels fairly brittle?

Reply:
I'll simplify back down to the equivalent `ContainSubstring` matcher for now to make it slightly less brittle.